### PR TITLE
Move cac validator to cac package

### PR DESCRIPTION
**Pull request by r0qs**
This PR continue the cac refactor and move the cac validation logic to the cac package.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -74,6 +74,7 @@ require (
 	google.golang.org/protobuf v1.25.0 // indirect
 	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
 	gopkg.in/ini.v1 v1.57.0 // indirect
+	gopkg.in/src-d/go-log.v1 v1.0.1
```
Comment (acud) [inline go.mod:77]: I am not sure this change is needed

**Review by acud — APPROVED** [review]
Very cool :clap: Thank you for the contribution!

File: go.mod (hunk 1 above)
Comment (r0qs) [reply go.mod:77]: Oh sorry, I guess this came with the merge with the master. I'll remove it.

**Review by janos — APPROVED** [review]
👍

**Pull request merged**